src/components/teaser-paid/teaser-paid.ts

```diff
@@ -0,0 +1,53 @@
+import { html, type CSSResultGroup, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import type { SbbChipElement } from '../chip';
+import { SbbLinkBaseElement } from '../core/common-behaviors';
+import type { SbbImageElement } from '../image';
+
+import style from './teaser-paid.scss?lit&inline';
+
+/**
+ * It displays an image and a chip with a text.
+ *
+ * @slot chip - Link content of the panel
+ * @slot image - The background image that can be a `sbb-image`
+ */
+@customElement('sbb-teaser-paid')
+export class SbbTeaserPaidElement extends SbbLinkBaseElement {
+  public static override styles: CSSResultGroup = style;
+
+  private get _chip(): SbbChipElement {
+    return this.querySelector('sbb-chip')!;
+  }
+
+  private get _image(): SbbImageElement | null {
+    return this.querySelector('sbb-image');
+  }
+
+  private _chipSlotChanged(): void {
+    if (this._chip) {
+      this._chip.color = 'charcoal';
+    }
```

Comment:
probably better to only set attribute here as it might be that chip is not already defined when the slotchange happens. Your opinion, @kyubisation ?

```suggestion
      this.querySelector('sbb-chip')?.setAttribute('color', 'charcoal');
```